src/components/PrePageTooltip/RowSections/index.tsx

```
@@ -456,3 +459,88 @@ export const BlockRowSections = (
+          <HashSpan
+            style={{ maxWidth: '20rem' }}
+            onClick={() => setShowTooltip(false)}
+          >
+            <span>
+              <strong>Properties: </strong>
+            </span>
+            <CenteredRow>
+              <PropertiesWrapper>
+                {Object.entries(sc?.properties || {}).map(([key, value]) =>
+                  value ? (
+                    <SpanWrapper>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)